docs/ref/class-based-views/mixins-single-object.txt

```diff
@@ -68,6 +68,10 @@ Single object mixins
 
         .. _insecure direct object reference: https://www.owasp.org/index.php/Top_10_2013-A4-Insecure_Direct_Object_References
 
+    .. method:: get_model()
+
```

Comment:
Add `.. versionadded:: 1.10`. Also add the method `docs/ref/class-based-views/flattened-index.txt`.